### PR TITLE
IP 14

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
Okay, so this was a crazy one. I tried to figure out where things are broken up, and I hope my notes in asciidoc are clear enough. I decided to leave them in there this time because it was easier to keep straight. I ended up having to write down on a piece of paper all the numbers and questions and answers and stuff in order so I could get a feel of how they are set up. Let me know if you want/need a pic of it, if it's still really confusing!! 😄 

Also, there are some answers that are multiple paragraphs. I wasn't sure if they all need a [.discourse-part] label....I can't remember if I've ever seen it in the past like that.

File: unabridged-works-volume-1/original/14-boston.adoc [hunk 1]
```diff
@@ -46,9 +46,10 @@ Even so Father; for so it seemed good in thy sight.
 Matt. 11:25-26
 ____
 
-=== TO THE RULERS, TEACHERS, AND PEOPLE OF New England
+[.blurb]
+=== To the Ruler New England
```
Comment (jaredh159) [inline unabridged-works-volume-1/original/14-boston.adoc:50]: Looks like we accidentally lost some of the words of this heading.

File: unabridged-works-volume-1/original/14-boston.adoc [hunk 2]
```diff
@@ -831,11 +858,15 @@ so encompassed with the walls and bulwarks of salvation,
 as that they feel no condemnation for sin, but a continual justification of the life;
 being taught, led, and enabled to walk, not after the flesh, but after the spirit. Rom. 8:1.
 
+//So it seems like this should be marked for the answer that follows, because without it, there are just two random answers in a row.
```
Comment (jaredh159) [inline unabridged-works-volume-1/original/14-boston.adoc:861]: Yeah, agreed, I think you did right here.

File: unabridged-works-volume-1/original/14-boston.adoc [hunk 3]
```diff
@@ -2119,12 +2200,19 @@ and not so much as an offence because of any difference
 (much less heart-burnings and persecutions);
 but a sweet waiting on the Lord for every one`'s growth in their several ranks and stations.
 
+====
+
 [.blurb]
 === Since my waiting on the LORD for the Presence and Guidance of his Spirit, in the examining the foregoing Grounds and Considerations, there came forth an appendix to JOHN NORTON`'s Book, wherein are laid down some further Grounds by way of justifying of their Proceedings; which, for their Sakes, and likewise on the Behalf of the Truth and People of GOD, I may also say somewhat to.
```
Comment (jaredh159) [inline unabridged-works-volume-1/original/14-boston.adoc:2206]: Can you get the 3 capitalized words in this blurb and change them to italics (with capitalized first letter)?

File: unabridged-works-volume-1/original/14-boston.adoc [hunk 4]
```diff
@@ -2254,6 +2348,9 @@ to which the spirit of the Lord alone can make them willing,
 nor to fall upon them because they were unwilling.
 This is from the dragon, wherever it is found.
 
+//the objection or questioin of for the next answer seems to start more at the end of this paragraph, and I wasn't sure what to do.
```
Comment (jaredh159) [inline unabridged-works-volume-1/original/14-boston.adoc:2351]: yeah, it's tough. i'm not sure what to do about it either, but i think your approach seems as good as any I can think of right now, so let's stick with it.

File: unabridged-works-volume-1/original/14-boston.adoc (hunk 1 above)
Comment (maijapapaya) [reply unabridged-works-volume-1/original/14-boston.adoc:50]: thanks for catching that, I probably got interrupted mid-sentence. 😆

**Comment by jaredh159** [comment]
Looks good, thanks!

**Pull request merged**